.travis.yml

```diff
@@ -82,6 +82,11 @@ script:
     if [[ -v MAVEN_CHECKS ]]; then
       ./mvnw install -DskipTests -B -T C1 -P ci
     fi
+  - |
+    if [[ -v BUILD_PRESTO_DOCKER ]]; then
+      PRESTO_VERSION=$(./mvnw --quiet --batch-mode --non-recursive  exec:exec -Dexec.executable='echo' -Dexec.args='${project.version}')
+      docker build . --build-arg "PRESTO_VERSION=${PRESTO_VERSION}" -t "presto:${PRESTO_VERSION}"
```

Comment:
can we run the docker you created and see if we can run simple `SELECT 1` or `SHOW CATALOGS`.

Reply:
I actually just tested this. `SHOW CATALOGS` and a `SELECT` against JMX both worked.

Reply:
Is this automated somehow? Can you please point a place where it is executed? I am sorry if I missed that.

Reply:
Oh, I'm sorry I think I misunderstood. You mean as part of the build process?

Reply:
Yes, just to catch the regression. You can do it as a follow up PR if you think it is going to be much work to do.

Reply:
@kokosing I've added this, however I'm not sure how it'll behave in Travis. Can you provide any insight?